pkg/registry/local.go

```diff
@@ -47,6 +48,13 @@ const (
 	LvscarePodFileName = "reg-lvscare.yaml"
 )
 
+type RegistryInfo struct { // nolint
+	ExternalRegistry *v2.ExternalRegistry
+	LocalRegistry    *v2.LocalRegistry
+	DeployHosts      []net.IP `json:"deployHosts,omitempty"`
+	Vip              string   `json:"vip,omitempty"`
```

Comment:
vip and DeployHosts only valid in LocalRegistry. so could we add it to `LocalRegistry`